backend/database/migrations/2021_12_10_023049_create_follows_table.php

```diff
@@ -16,7 +16,7 @@ public function up()
         Schema::create('follows', function (Blueprint $table) {
             $table->id();
             $table->foreignId('follower_id');
-            $table->foreignId('followed_id');
+            $table->foreignId('following_id');
```

Comment:
**Do not edit the migrated migrations**, this works if you run `php artisan migrate:fresh` yet is a bad practice, it flushes/resets all the data when running `migrate:fresh`.

When updating/deleting a column in a table create new migration file, so that the person who will check(QA) or programmers will run only `php artisan migrate` and the data will stay the same.

Related video: https://www.youtube.com/watch?v=ArNb3xMY-Gc